### PR TITLE
[FEAT] Don't put your business logic into controllers

### PR DESCRIPTION
**Pull request by gabriel-rcpereira**
- Adding a new tip describing a better approach to hold your business logic encapsulated;

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,56 @@ public class PersonController {
 }
 ```
 
+* Simplify your controller keeping it fin
```
Comment (helpermethod) [inline README.md]: What is meant by `fin` :)?

Comment (gabriel-rcpereira) [reply README.md]: It's just a typo 🤦 

I fixed it.

Thanks!

**Comment by helpermethod** [comment]
Looks great, will have another look at it this evening!

**Comment by helpermethod** [comment]
LGTM! Thank you very much!

**Pull request merged**